codec2/msdk-codec2/init.te

```diff
@@ -0,0 +1,5 @@
+#
+# init
+#
+
+allow init vendor_file:file execute;
```

Comment:
Why add this rule? what's the relevant avc eror for this?

Reply:
Together with above waring, below message is shown.

11-23 15:28:09.318   193   566 W libc    : Unable to set property "ctl.interface_start" to "android.hardware.media.c2@1.0::IComponentStore/default": error code: 0      x20
 2760 11-23 15:28:09.321   193   566 I hwservicemanager: Tried to start android.hardware.media.c2@1.0::IComponentStore/default as a lazy service, but was unable to. Usu      ally this happens when a service is not installed, but if the service is intended to be used as a lazy service, then it may be configured incorrectly.

Reply:
Where is hardware.intel.media.c2@1.0-service?

Reply:
It is under /vendor/bin/hw/